### ISSUE TITLE
Can't round or format numbers in FeatureInfo templates

### ISSUE THREAD

**Issue by davekennewell** [GeoNode/geonode#6469]
## Expected Behavior
When reading an attribute value in a FeatureInfo template, it was expected to be possible to format these values using FreeMarker syntax. For example test1 here should return `1234`:

![image](https://user-images.githubusercontent.com/9131321/94691907-f5b5f680-0342-11eb-8c6a-5d58ec1e64d4.png)

## Actual Behavior
test1 returns nothing at all:
![image](https://user-images.githubusercontent.com/9131321/94692144-40377300-0343-11eb-96fb-f99f0a687e87.png)

## Steps to Reproduce the Problem

  1. The screenshots above were generated on master.demo.geonode.org
  2. The tests are available to check here: https://master.demo.geonode.org/layers/geonode:san_andres_y_providencia_coastline4

## Specifications

  - master.demo.geonode.org

**Comment by gannebamm** [comment]
Maybe this is related to the following not enabled:

https://docs.geoserver.org/latest/en/user/tutorials/GetFeatureInfo/html.html#accessing-static-methods

**Comment by t-book** [comment]
@davekennewell Did you ever find an answer?

**Comment by gannebamm** [comment]
> Maybe this is related to the following not enabled:
> 
> https://docs.geoserver.org/latest/en/user/tutorials/GetFeatureInfo/html.html#accessing-static-methods

I tried that and it did *not* work on my instance (master)

**Comment by davekennewell** [comment]
Nope - I never found an answer for this. I did try `-Dorg.geoserver.htmlTemplates.staticMemberAccess=*` from the geoserver docs pretty extensively, and it did not work for me. I'd still be interested in a solution, but I don't know where to look, it seems to be a geoserver internal thing, right?

**Comment by giohappy** [comment]
There's a misunderstanding here. The HTML template that you configure in GeoNode is a [MapStore template](https://mapstore.readthedocs.io/en/latest/user-guide/layer-settings/#templates), which only supports attributes interpolation, i.e. `${properties.NAME_OF_THE_FIELD}`.

MapStore itself offers the option to select the HTML GetFeatureInfo response (from Geoserver), but at the moment this option is not exposed inside GeoNode client.

Basically, what we have now, is the following option preconfigured (this screenshot is taken from [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/), not GeoNode).

![image](https://user-images.githubusercontent.com/571129/150947324-027e0d19-9cb8-405f-8413-3436d8713dc7.png)

**Comment by davekennewell** [comment]
Hi @giohappy - thanks, that explaination makes sense. To clarify the issue, the mapstore client has always rendered the template set in GeoNode - what I was looking for was a way to round the float numbers that are returned in the geonode/mapstore HTML template. For example, the Shape_Leng attribute here would need to be rounded to 2 decimal places (screenshot from geonode): 

![image](https://user-images.githubusercontent.com/9131321/150949641-335afa15-f507-460e-b405-eb3431b8d9b2.png)

**Comment by giohappy** [comment]
Unfortunately this isn't supported at the moment in MapStore @davekennewell. 

The only alternative, without any development effort, is to publish a Geoserver view of your data, where you can use SQL formatting, but I know it's quite a convoluted solution.

Options to be considered (but development is required) are:
 - expose the configuration to select Geoserver HTML as the GFI output for a layer
 - extend MapStore templates to support formatting

**Comment by davekennewell** [comment]
OK, thanks @giohappy - now I understand where/how it could be implemented... I was a bit confused for a while there :).
